renderer/index.js

```diff
@@ -595,6 +597,10 @@ function addTorrentEvents (torrent) {
     var torrentSummary = getTorrentSummary(torrent.infoHash)
     torrentSummary.status = 'seeding'
     updateTorrentProgress()
+    torrent.getFileModtimes(function (err, fileModtimes) {
+      if (err) return onError(err)
+      torrentSummary.fileModtimes = fileModtimes
```

Comment:
https://github.com/feross/webtorrent-desktop/blob/78b90296c0adad35b53b83c7aa960b0e2dfcd674/main/tray.js#L49

On Windows and Linux, the only way to quit the app is from the tray icon.

This is broken on Mac, where quitting works differently. I'll add an event handler to fix it.